README.md

```diff
@@ -1,5 +1,4 @@
 [![Build Status](https://travis-ci.org/owncloud/owncloud-sdk.svg?branch=master)](https://travis-ci.org/owncloud/owncloud-sdk)
```

Comment:
travis CI ref should also be removed, since the last build ran 3 years ago.

Reply:
Aye 👀